Chocolatey.Cake.Recipe/Content/sonarqube.cake

```diff
@@ -31,6 +31,13 @@ BuildParameters.Tasks.InitializeSonarQubeTask = Task("Initialize-SonarQube")
         SonarQubeSettings.Url = BuildParameters.SonarQubeUrl;
     };
 
+    if (BuildParameters.ShouldRunDependencyCheck)
+    {
+        SonarQubeSettings.ArgumentCustomization = args => args
+            .Append(string.Format("/d:sonar.dependencyCheck.jsonReportPath={0}", BuildParameters.RootDirectoryPath.CombineWithFilePath("dependency-check-report.json")))
+            .Append(string.Format("/d:sonar.dependencyCheck.htmlReportPath={0}", BuildParameters.RootDirectoryPath.CombineWithFilePath("dependency-check-report.html")));
```

Comment:
Would it make sense for these to be placed in a consistent output location?  Rather than at the root of the project?  We have a `TestResults` and `TestCoverage` folder in the `code_drop` folder, so we could use those, or create a new folder in this location.

Either way, I think adding an entry in the `paths.cake` file, similar to `testCoverageOutputFilePath` would make sense here, so that things are defined in one place.

Reply:
That makes perfect sense, and is a lot cleaner (both code wise, but also files on disk).

I think I've got this complete in a second commit. Please do let me know if I missed anything.